src/main.browser.ts

```diff
@@ -42,20 +44,28 @@ bootstrap(App, [
   runEffects(effects),
 
   /**
-   * provideRouter sets up all of the providers for @ngrx/router. It accepts
+   * provideRouter sets up all of the providers for @angular/router. It accepts
    * an array of routes and a location strategy. By default, it will use
    * `PathLocationStrategy`.
-   *
-   * Source: https://github.com/ngrx/router/blob/master/lib/index.ts#L44-L51
    */
-  provideRouter(routes, HashLocationStrategy),
+  provideRouter(routes),
+
+  /**
+   * Make router directives available to all components
+   */
+  { provide: PLATFORM_DIRECTIVES, useValue: [ROUTER_DIRECTIVES], multi: true },
+
+  /**
+   * Override the default location strategy with `HashLocationStrategy`
+   */
+  { provide: LocationStrategy, useClass: HashLocationStrategy },
 
   /**
    * connectRouterToStore configures additional providers that synchronize
    * router state with @ngrx/store. This lets you debug router state using
    * ngrx/store and to change the location by dispatching actions.
    */
-  connectRouterToStore(),
+  // connectRouterToStore(),
```

Comment:
I would rather remove references to @ngrx/store-router until we get the bindings working for @angular/router